trie/committer.go

```diff
@@ -232,3 +222,18 @@ func estimateSize(n node) int {
 		panic(fmt.Sprintf("node type %T", n))
 	}
 }
+
+// wrapDeletions puts all tracked deletions into the dirty nodeset.
+func wrapDeletions(set *NodeSet, tracer *tracer) {
```

Comment:
Could you change the signature a bit? Then the two call-sites could look like this, respectively: 

```
c.tracer.HandleDeletions(c.nodes)
```
and
``` 
t.tracer.HandleDeletions(set)
```
Seems odd to me that you made it into a standalone method like that.. (?)

Reply:
Fixed